tests/test_edgeql_expressions.py

```diff
@@ -5941,6 +5941,238 @@ async def test_edgeql_expr_range_31(self):
                     )
                 ''')
 
+    async def test_edgeql_expr_range_32(self):
```

Comment:
I don't see any error cases. What happens if I pass various kinds of garbage to `select <range<cal::local_date>>to_json($1)`? Think missing some attributes, or an outright wrong kind of JSON (array, number 42, object literal with all wrong keys, etc)

Reply:
I've added a bunch more tests for various wrong JSON inputs.